core/src/main/java/org/axonframework/queryhandling/backpressure/WindowBackpressure.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2010-2018. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
```

Comment:
This copyright notice doesn't have the additional empty lines! :o

Reply:
This one is after I've updated copyright ;), from now on I wouldn't expect additional lines :)